lib/stylesheet/importer.rb

```diff
@@ -40,6 +40,35 @@ def self.register_imports!
         end
       end
 
+      register_import "font" do
+        font = DiscourseFonts.fonts.find { |f| f[:key] == SiteSetting.base_font }
+
+        contents = <<~EOF
+          #{font_css(font)}
+
+          :root {
+            --font-family: #{font[:name]};
```

Comment:
Previously we had `Helvetica, Arial, sans-serif` for the font-family, because Helvetica is not available by default in Windows machines, so the default font would fall back to Arial there. I wonder whether we should include the fallbacks, at least for the default, non-custom font.